client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
Submission Details page
![Screenshot of Submission Details page showing 14px font size](https://github.com/MESH-Research/Pilcrow/assets/2668987/77b07d79-402e-4389-b0a0-e32612868a68)

Reply:
These all appear as 16px.